numpy/__init__.pyi

```diff
@@ -3367,7 +3367,7 @@ class busdaycalendar:
     def __new__(
         cls,
         weekmask: ArrayLike = ...,
-        holidays: ArrayLike = ...,
+        holidays: ArrayLike | dt.date | _NestedSequence[dt.date] = ...,
```

Comment:
why not create a type alias for `ArrayLike | dt.date | _NestedSequence[dt.date]` ?

Reply:
Considering they're only applicable for very limited (and niche) set of functions, I'm not sure it's worthwhile to introduce aliases here. Line length is also not an issue here.